### DISCUSSION TITLE
Travis fold equivalent

### DISCUSSION THREAD

**Discussion by fr4ngus** [community/community#26448] [Actions]
<p>Hi,</p>
<p>I want to know if the CI has a Travis Fold equivalent ?</p>
<p>Currently, I’m using:</p>
<pre><code>echo ::group::Group1
echo ::endgroup::
</code></pre>
<p>But it is not mentionned in the doc and I can’t create imbricate groups.</p>
<p>Thank you for your support.</p>

**Comment by mscoutermarsh** [comment]
<p>Hi <a class="mention" href="/fr4ngus">@fr4ngus</a>,</p>
<p>Here’s some information on the group command: <a href="https://github.com/actions/toolkit/blob/df7e2c13c89ea93069cad62f1adfe78a87111cc0/docs/commands.md#group-and-ungroup-log-lines" rel="nofollow noopener">https://github.com/actions/toolkit/blob/df7e2c13c89ea93069cad62f1adfe78a87111cc0/docs/commands.md#group-and-ungroup-log-lines</a></p>
<p>I’m not familiar with Travis Fold’s. Does this cover the info you were looking for? </p>

**Comment by fr4ngus** [comment, marked as answer]
<p>Hi,</p>
<p>Yes, but it doesn’t support subfold structure.</p>
<p>You have an example in Travis CI : <a href="https://travis-ci.org/macbre/travis-fold" rel="nofollow noopener">https://travis-ci.org/macbre/travis-fold</a></p>
<p>Do you know if it is possible ?</p>
<p>Thank you</p>
